<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="menu1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Giám sát</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="menu2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Giám sát vào ra</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="menu3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Thêm sinh viên</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="menu4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Nhiệt độ và độ ẩm</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="menu5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Thoát</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>1021</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="photoButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);
border-color: rgb(8, 8, 8);
border-color: rgb(0, 0, 0);
border-radius: 30</string>
    </property>
    <property name="text">
     <string>Chụp ảnh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
